.github/workflows/ubuntu.yml

```diff
@@ -3,8 +3,7 @@ name: Ubuntu CI
 on:
   workflow_dispatch:
   push:
-    branches:
-      - master
+    branches: [master, dev]
```

Comment:
@ssheorey do we want this addition?

Reply:
Can revert it,  no problem. It's just for testing